src/api/sso/test/row-level-policies.test.js

```diff
@@ -0,0 +1,350 @@
+const { hash } = require('@senecacdot/satellite');
+const { createClient } = require('@supabase/supabase-js');
+
+const { SUPABASE_URL, ANON_KEY } = process.env;
+const {
+  ensureUsers,
+  createTelescopeUsers,
+  cleanupTelescopeUsers,
+  addFeeds,
+  cleanFeeds,
+  ensureFeeds,
+} = require('./supabaseUtil');
+const { createToken } = require('../src/token');
+
+const supabase = createClient(SUPABASE_URL, ANON_KEY);
```

Comment:
I wonder if it would be better to create this client in your tests, or `beforeEach` so it's more clear who you are when making the requests below?  I had to think about it as I read.